packages/main/src/themes/Button.css

```diff
@@ -7,6 +7,7 @@
 :host {
 	min-width: var(--_ui5_button_base_min_width);
 	height: var(--_ui5_button_base_height);
+	line-height: 100%;
```

Comment:
Went for 100% because it's directly based to the font family, but after second consideration, this may result in unexpected looks for the button, so I agree on going on normal as value.